torch/csrc/jit/tensorexpr/graph_opt.cpp

```diff
@@ -206,6 +206,56 @@ std::shared_ptr<Graph> removeUnusedSelfArgument(
   return graph;
 }
 
+std::vector<int64_t> makeShapesSymbolic(
+    std::shared_ptr<Graph>& graph,
+    const std::vector<int64_t>& size_vals) {
+  std::unordered_set<Value*> values;
```

Comment:
We should probably consider exposing `all_values` from `Graph`: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/ir/ir.h#L1160

Reply:
I think it's a relatively rare case when we need to iterate over all values (and not nodes), so I'm not sure it's worth exposing (also, I think a common pattern "for each graph input, then for each input and output of each graph node, then for each graph output" works well for most cases and frankly could've been used here as well).